demo/psa/prov-verif-e2e/COCLI_README.md

```diff
@@ -0,0 +1,61 @@
+# Provisioning Instructions
+
+Follow below instructions if one wishes to create new Concise Module Identifiers (CoMID's) and use them in creating Concise Reference Integrity Manifests (CoRIM's) using reference templates located under `docs/demo/psa/prov-verif-e2e/data/templates` to provision them in Veraison Verification Service.
+
+More details about CoRIM and CoMID can be found [here](datatracker.ietf.org/doc/draft-birkholz-rats-corim/)
+
+## Initial Setup
+
+Do this ONLY if it is not done already:
+
+In a new bourne shell session
```

Comment:
```suggestion
* Commands below assume execution in a Bourne-compatible shell. Please adjust appropriately in case any other shell is used.

In a new shell session
```